test/python/test_transport.py

```diff
@@ -539,3 +541,118 @@ def test_set_customary_units(self, gas):
         tr2 = gas.species('N2').transport
         assert tr1.dispersion_coefficient == approx(tr2.dispersion_coefficient)
         assert tr1.quadrupole_polarizability == approx(tr2.quadrupole_polarizability)
+
+class TestHighPressureGasTransport():
+    """
+    Note: to re-create the reference file:
+    (1) Set PYTHONPATH to build/python.
+    (2) Go into test/python directory and run:
+        pytest --save-reference=high_pressure_transport test_transport.py::TestHighPressureGasTransport::test_high_pressure_transport
+        pytest --save-reference=high_pressure_chung_transport test_transport.py::TestHighPressureGasTransport::test_high_pressure_chung_transport
+    (3) Compare the reference files created in the current working directory with
+        the ones in test/data and replace them if needed.
+    """
+
+    def test_failure_for_species_with_no_properties(self, test_data_path):
+        """
+        All species must have critical properties defined to use the high pressure
+        transport model. This test uses a YAML file with a specified value of the
+        a and b parameters for the Peng-Robinson equation of state, which should
+        be parsed by the thermo model and will set the critical properties to
+        non-physical values. These non-physical values should trigger an error
+        in the high-pressure transport model.
+        """
+        gas = ct.Solution(test_data_path / 'methane_co2_noCritProp.yaml')
```

Comment:
The test data path is automatically added to the Cantera search path, so you can just write:
```suggestion
        gas = ct.Solution('methane_co2_noCritProp.yaml')
```
and remove the fixture from the method.

The case where you need `test_data_path` is for accessing files that aren't read by Cantera's AnyMap/YAML functions.